cmd/minishift/main.go

```diff
@@ -18,8 +18,10 @@ package main
 
 import (
 	"github.com/minishift/minishift/cmd/minishift/cmd"
+	"github.com/minishift/minishift/pkg/minikube/machine"
 )
 
 func main() {
+	machine.StartDriver()
```

Comment:
>  review itself was messy as it contained incomplete merge actions

not sure what you mean?

> just wonder how this made things still work on macOS.

just to be clear, this method does by no means "start" anything as the method name makes you think. The interesting bit is:

    localbinary.CurrentBinaryIsDockerMachine = true


The if statement is pretty much useless and only comes into play in the test of this method. I was experimenting with this to better understand it and removed it w/i bringing it back. Never noticed, since at least for OS X, it does not make a difference. I have a theory mourned this, but it is not well formed yet.